frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1307,19 +1307,21 @@ const ruleHasAlertState = (rule: Rule, state: AlertStates): boolean =>
 const ruleAlertStateFilter = (filter, rule: Rule) =>
   _.some(rule.alerts, (a) => filter.selected.has(a.state)) || _.isEmpty(filter.selected);
 
+export const AlertStateFilter: RowFilter = {
```

Comment:
Please use lowercase a for variables (which are not a react component)

```suggestion
export const alertStateFilter: RowFilter = {
```